pkg/session/session.go

```diff
@@ -223,6 +223,9 @@ type session struct {
 	sandBoxMode bool
 
 	cursorTracker cursor.Tracker
+
+	// Used to wait for all async commit background jobs to finish.
+	asyncCommitWaitGroup sync.WaitGroup
```

Comment:
May just name it `commitWaitGroup`, because not only async-commit txns are tracked.